admin/services/nca-tools-auth.service.js

```diff
@@ -58,7 +58,7 @@ const service = {
 module.exports = service
 
 /**
- * Verify the provided signature is valid.
+ * Verify the provided signature is valid. - commented out at the moment, see the check on line 25
```

Comment:
we do need to come back to this and discuss with 3rd party (we have a backlog item for it).